### PR TITLE
8214733: runtime/8176717/TestInheritFD.java timed out

### PR DESCRIPTION
**Pull request by hseigel**
Please review this small change to fix TestInheritFD.java.  The logs in the JBS bug indicate that the test probably timed out waiting for the lsof command to finish.  This fix adds the -a and +d <dir> options to the lsof command to limit its scope.  The +d option tells lsof to only look in the specified directory for open files.  The -a option combines the -p <pid> and the +d option.  (An alternative fix would be to use the -b option.)

The fix was tested by running the test 1000 times on Linux x64 and Mac OS aarch64.  The fix was also sanity tested on Linux aarch64, Mac OS x64, and windows.

Thanks, Harold

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8214733](https://bugs.openjdk.java.net/browse/JDK-8214733): runtime/8176717/TestInheritFD.java timed out

### Reviewers
 * [Leonid Mesnik](https://openjdk.java.net/census#lmesnik) (@lmesnik - **Reviewer**) ⚠️ Review applies to db0325d08054fa613e707a28d9d38f2c051b6bdc
 * [Coleen Phillimore](https://openjdk.java.net/census#coleenp) (@coleenp - **Reviewer**) ⚠️ Review applies to db0325d08054fa613e707a28d9d38f2c051b6bdc
 * [Daniel D. Daugherty](https://openjdk.java.net/census#dcubed) (@dcubed-ojdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7179/head:pull/7179` \
`$ git checkout pull/7179`

Update a local copy of the PR: \
`$ git checkout pull/7179` \
`$ git pull https://git.openjdk.java.net/jdk pull/7179/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7179`

View PR using the GUI difftool: \
`$ git pr show -t 7179`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7179.diff">https://git.openjdk.java.net/jdk/pull/7179.diff</a>

</details>

### CODE REVIEW

**Review by lmesnik — APPROVED** [review]

**Review by coleenp — APPROVED** [review]
Nice.

File: test/hotspot/jtreg/runtime/8176717/TestInheritFD.java [hunk 1]
```diff
@@ -187,10 +188,9 @@ static void fakeLeakyJVM(boolean fake) {
 
     static Collection<String> outputContainingFilenames() {
         long pid = ProcessHandle.current().pid();
-
         String[] command = lsofCommand().orElseThrow(() -> new RuntimeException("lsof like command not found"));
-        System.out.println("using command: " + command[0] + " " + command[1]);
-        return run(command[0], command[1], "" + pid).collect(toList());
+        System.out.println("using command: " + command[0] + " -a +d " + USER_DIR + " " + command[1] + " " + pid);
+        return run(command[0], "-a", "+d", USER_DIR, command[1], "" + pid).collect(toList());
```
Comment (dcubed-ojdk) [inline test/hotspot/jtreg/runtime/8176717/TestInheritFD.java:193]: You might want to add a comment above L192:
// Only search the directory in which the VM is running (user.dir property):

**Review by dcubed-ojdk — APPROVED** [review]
Thumbs up. Minor request for a comment.

**Review by dcubed-ojdk — APPROVED** [review]
Thanks for adding the comment.

Since this failure mode reproduces on my Ubuntu 20.04 machine, I'll
keep my eyes open after you integrate.

**Comment by hseigel** [comment]
Thanks Leonid, Coleen, and Dan for the reviews!

/integrate

**Pull request closed without merging**